generators/app/templates/bin/setup

```diff
@@ -1,34 +1,14 @@
-#!/usr/bin/env ruby
-require 'pathname'
-require 'fileutils'
-include FileUtils
+#!/bin/bash
+function realpath { echo $(cd $(dirname $1); pwd)/$(basename $1); }
+SCRIPT=`realpath $0`
+ROOT_PATH="$(dirname $SCRIPT)/../"
 
-# path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+echo "Server setup..."
+cd "$ROOT_PATH/server" && bin/setup
 
-def system!(*args)
-  system(*args) || abort("\n== Command #{args} failed ==")
-end
+wait
 
-chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file.
+echo "Client setup..."
+cd "$ROOT_PATH/client" && npm install
```

Comment:
hmm, didn't thin in that case, but i think that should be handled with the `bin/setup`